ci_build_images/opensuse15.Dockerfile

```diff
@@ -0,0 +1,50 @@
+#
+# Buildbot worker for building MariaDB
+#
+# Provides a base OpenSUSE image with latest buildbot worker installed
+# and MariaDB build dependencies
+
+ARG base_image
+FROM "$base_image"
+LABEL maintainer="MariaDB Buildbot maintainers"
+
+# Install updates and required packages
+RUN zypper update -y && \
+    zypper install -y -t pattern devel_basis && \
+    zypper install -y git ccache subversion \
+    python-devel libffi-devel openssl-devel glibc-locale\
+    python-pip curl wget && \
+    # install MariaDB dependencies
+    #zypper mr -er repo-source
+    zypper -n si -d mariadb
+
+# Create buildbot user
+RUN useradd -ms /bin/bash buildbot && \
+    mkdir /buildbot && \
+    chown -R buildbot /buildbot && \
+    curl -o /buildbot/buildbot.tac https://raw.githubusercontent.com/MariaDB/mariadb.org-tools/master/buildbot.mariadb.org/dockerfiles/buildbot.tac
+WORKDIR /buildbot
```

Comment:
This part shouldn't be necessary. Any changes should be defined in common.Dockerfile. The final dockerfile used for building the image is the concatenation between the opensuse15.Dockerfile and common.Dockerfile. See https://github.com/MariaDB/buildbot/blob/de9660be1e0bbfe608f23a27560ae4d0d9857431/.github/workflows/bb_containers.yml#L156